src/pages/directory.js

```diff
@@ -0,0 +1,34 @@
+import { Inter } from 'next/font/google'
```

Comment:
No need to load all of this in to the file as is already loaded in the app loader